platform/darwin/src/MGLAccountManager.h

```diff
@@ -1,11 +1,12 @@
-#import <Foundation/Foundation.h>
+#import "MGLTypes.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
  The MGLAccountManager object provides a global way to set a Mapbox API access
  token.
  */
+MGL_EXTERN
```

Comment:
If I’m not mistaken, placing this macro on a separate line between the comment block and the signature breaks Quick Help, which only looks one line up to find a comment block. Would this macro work if we stick it above the comment block? We should also make sure jazzy – which like Quick Help also relies on Clang to parse out comment blocks – is OK with whatever we end up with.